code/game/turfs/turf.dm

```diff
@@ -118,7 +118,8 @@ GLOBAL_LIST_EMPTY(station_turfs)
 			T.multiz_turf_new(src, UP)
 
 	// by default, vis_contents is inherited from the turf that was here before
-	vis_contents.Cut()
+	if (length(vis_contents))
```

Comment:
This is the one line that saved like 0.7s

Reply:
here too then